CHANGES.md

```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+## 1.2.4
```

Comment:
FYI the version here doesn't match the version in pack.yaml :( sorry!!!

```suggestion
## 1.3.0
```
You should be able to accept this suggestion and have it auto-commit the change.